app/models/goal.rb

```diff
@@ -1,4 +1,5 @@
 class Goal < ApplicationRecord
+  belongs_to :user
```

Comment:
missing spec for this association

Reply:
oh thank you for the review, i added it.